### PR TITLE
fix: ksql grb Postinfo column all values

### PR DESCRIPTION
**Pull request by jvandaal**

### CODE REVIEW

File: .ksql/03_POSTAL_SNAPSHOT_OSLO_STREAM_FLATTEN_GRB.ksql [hunk 1]
```diff
@@ -6,13 +6,13 @@ CREATE OR REPLACE STREAM IF NOT EXISTS postal_snapshot_oslo_stream_flatten_grb
       IDENTIFICATOR->ID Id,
       IDENTIFICATOR->VERSIEID Versieid,
       IDENTIFICATOR->OBJECTID Postcode,
-      IFNULL(GEMEENTE->OBJECTID, '') Niscode,
+      IFNULL(GEMEENTE->OBJECTID, '') Niscode, Postinfo,
```
Comment (lvermeulen) [inline .ksql/03_POSTAL_SNAPSHOT_OSLO_STREAM_FLATTEN_GRB.ksql:9]: forgot a carriage return?

Comment (ArneD) [inline .ksql/03_POSTAL_SNAPSHOT_OSLO_STREAM_FLATTEN_GRB.ksql:9]: `NisCode, Postinfo` probably postinfo not needed here

Comment (jvandaal) [reply .ksql/03_POSTAL_SNAPSHOT_OSLO_STREAM_FLATTEN_GRB.ksql]: Fixed

Comment (jvandaal) [reply .ksql/03_POSTAL_SNAPSHOT_OSLO_STREAM_FLATTEN_GRB.ksql]: Fixed

**Review by lvermeulen — CHANGES_REQUESTED** [review]
inline

**Review by lvermeulen — APPROVED** [review]

**Pull request merged**